exec/supervisor/config.go

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2019 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package supervisor
+
+import "github.com/pkg/errors"
+
+// Config represents configuration file for the supervisor procedure
+type Config struct {
+	// The configuration file consists from a list of processes
+	Processes []Process `json:"processes"`
+}
+
+// Process is a single process instance started by the supervisor
+type Process struct {
```

Comment:
I think this should rather be named `Program` to avoid disambiguation with `processmanager.Process`.